rasa_nlu/train.py

```diff
@@ -62,6 +62,21 @@ def init():     # pragma: no cover
     return config
 
 
+def do_train_in_worker(config):
```

Comment:
Ok, I think I missed your point there: could you explain a bit further this metacopy stuff? 😄